site/templates/tv/detail.jet

```diff
@@ -29,7 +29,7 @@
             <s72-image src="{{tvseason.Images.Portrait}}" alt="{{tvseason.Title}}" class="poster poster-image poster-image-portrait"></s72-image>
           </div>
         {{end}}
-        <div class="meta-detail-content">
+        <div class="meta-detail-content meta-detail-info-width">
```

Comment:
Is this necessary if tv detail doesn't have the sponsor image?

Also... should the tv season and bundle pages have sponsor image support given that they will have access to the same custom fields (I wish this wasn't done with custom fields).

Reply:
Yes unfortunately. Reason being the width 563px and 1100px media queries can't be on the film detail page meta-detail-content div any more (as it wraps sponsor). (Note as mentioned .meta-detail-info-width has been removed in favor of targeted css.)

Nope, acceptance criteria justified around film and custom fields only. I think clients wanted to set sponsor text per film detail